internal/handlers/handler_test.go

```diff
@@ -122,6 +122,26 @@ func TestFind(t *testing.T) {
 	// Do not use sentences, spaces, or underscores in subtest names
 	// to make it easier to run individual tests with `go test -run test/name` and for consistency.
 	testCases := map[string]testCase{
+		"DotNotationQuery": {
```

Comment:
We started moving away from those tests to integration tests that test both FerretDB and MongoDB. Please add a new test there instead: https://github.com/FerretDB/FerretDB/blob/c4d88cfd9d5c03ffc48026f59116aec5be2a6c84/integration/query_comparision_test.go#L42-L49